Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -11,6 +11,8 @@
 from Tests.scripts.utils import logging_wrapper as logging
 from Tests.Marketplace.marketplace_constants import GCPConfig
 
+XSIAM = "marketplacev2"
+XSOAR = "xsoar"
```

Comment:
We have constants for these. 
MarketplaceVersions.MarketplaceV2
MarketplaceVersions.XSOAR

Reply:
https://github.com/demisto/content/blob/4985f10051d8f587fe7249358e86aa41bf3dd2a6/Tests/Marketplace/marketplace_constants.py#L25 
Still have constants